library/src/main/java/com/github/clans/fab/FloatingActionMenu.java

```diff
@@ -280,9 +282,11 @@ private void createMenuButton() {
 
         mImageOpenButton = new ImageView(getContext());
         mImageOpenButton.setImageDrawable(mOpenIcon);
+        mImageOpenButton.setId(mOpenIconResId);
```

Comment:
@jmorales4 - Ids are set here and 3 other places below.

Reply:
mOpenIconResId is guaranteed to be 0 here